x/crosschain/keeper/cctx_orchestrator_validate_inbound.go

```diff
@@ -48,3 +56,44 @@ func (k Keeper) ValidateInbound(
 
 	return &cctx, nil
 }
+
+// CheckIfMigrationDeposit checks if the sender is a TSS address and returns an error if it is.
+// If the sender is an older TSS address, this means that it is a migration transfer, and we do not need to treat this as a deposit and process the CCTX
+func (k Keeper) CheckIfMigrationDeposit(ctx sdk.Context, msg *types.MsgVoteInbound) error {
+	additionalChains := k.GetAuthorityKeeper().GetAdditionalChainList(ctx)
+
+	historicalTssList := k.zetaObserverKeeper.GetAllTSS(ctx)
+	chain, found := k.zetaObserverKeeper.GetSupportedChainFromChainID(ctx, msg.SenderChainId)
+	if !found {
+		return observertypes.ErrSupportedChains.Wrapf("chain not found for chainID %d", msg.SenderChainId)
+	}
+
+	if chain.CctxGateway != chains.CCTXGateway_observers {
+		return nil
+	}
+
+	for _, tss := range historicalTssList {
+		if chains.IsEVMChain(chain.ChainId, additionalChains) {
+			ethTssAddress, err := crypto.GetTssAddrEVM(tss.TssPubkey)
+			if err != nil {
+				return errors.Wrap(types.ErrInvalidAddress, err.Error())
+			}
```

Comment:
I dont mind changing to continue , but there are the points I considered 
- I cant think of a situation this would fail , if it was passing previously 
- If in case this fails, and the transfer is actually a migration tx , this would end up minting the tokens on zevm

Reply:
It would be more precautious as we already had a case before when inbound were block because of two restrictive checks. I may also think the error would not allow the deposit to be reverted which might cause loss of funds